### PR TITLE
Route demo

### PR DESCRIPTION
**Pull request by Stockfoot**
Full modularization
Routes should be complete, unless you think of some others
No errors and no white screens of death

### CODE REVIEW

File: inc/section_05_permissions.inc [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
-function section_05_permissions() {
-  return [
+function section_05_permissions($checklist_routes) {
 
-  // Section 5 User Permissions.
+  return [
+    // Section 5 User Permissions.
     '#title' => t('User Permissions'),
     '#description' => t('<p>User Permission settings for the Drupal site.</p>'),
     'manager_permissions' => [
```
Comment (thejimbirch) [inline inc/section_05_permissions.inc:19]: Manager?

File: inc/section_03_forms.inc [hunk 2]
```diff
@@ -19,15 +19,15 @@ function section_03_forms() {
       '#description' => t('Check all webforms for recipient email and test.'),
       'handbook_page' => [
         '#text' => t('Webform'),
-        '#url' => $webform_route,
+        '#url' => $checklist_routes["webform_route"],
```
Comment (thejimbirch) [inline inc/section_03_forms.inc:22]: Can we change this to `webform` instead of `webform_route`?

File: inc/section_08_security.inc [hunk 3]
```diff
@@ -8,136 +8,136 @@
...
+function section_08_security($checklist_routes) {
 
   return [
     // Section 8 Performance and Security.
-    '#title' => t('Performance and Security'),
-    '#description' => t('<p>Performance and Security settings for the Drupal site.</p>'),
-    'page_caching' => [
-      '#title' => t('Page Caching'),
-      '#description' => t('Ensure that page caching is enabled.'),
+    '#title'            => t('Performance and Security'),
```
Comment (thejimbirch) [inline inc/section_08_security.inc:17]: All of this is odd formatting

Comment (thejimbirch) [reply inc/section_08_security.inc:17]: line 17-20 then line 24 has one extra space.

Comment (thejimbirch) [reply inc/section_08_security.inc:17]: Actually same through the rest of the file

File: inc/section_12_drupal_modules.inc [hunk 4]
```diff
@@ -18,23 +20,18 @@ function section_12_drupal_modules() {
...
-      // '#text' => t( 'Google Analytics module' ),
-      // '#url' => $google_anaytics_route,
+      'handbook_page' => [
+        '#text' => t('Coder module'),
+        '#url' => Url::fromUri('https://www.drupal.org/project/coder'),
+      ],
     ],
     'devel_module' => [
       '#title' => t('Devel Module'),
       '#description' => t('Ensure that the Devel module is disabled on the production site.'),
```
Comment (thejimbirch) [inline inc/section_12_drupal_modules.inc:34]: You could like to the modules page.

File: inc/section_05_permissions.inc (hunk 1 above)
Comment (thejimbirch) [inline inc/section_05_permissions.inc:19]: Yeah, this is correct, deleting

File: inc/section_12_drupal_modules.inc (hunk 4 above)
Comment (thejimbirch) [reply inc/section_12_drupal_modules.inc:34]: You could _link_ to the modules page.

File: inc/section_03_forms.inc (hunk 2 above)
Comment (Stockfoot) [reply inc/section_03_forms.inc:22]: Changed to `webform`

File: inc/section_12_drupal_modules.inc (hunk 4 above)
Comment (Stockfoot) [reply inc/section_12_drupal_modules.inc:34]: That was my original thought, but then I read "Coder is not a module, it is used as command line tool and in IDEs." on the drupal.org page

**Pull request merged**